.travis.yml

```diff
@@ -59,8 +59,7 @@ after_failure:
 
 branches:
   only:
-    - master
-    - maintable-beta
+    - fix-for-issue-3959
```

Comment:
Is there a reason why you disabled travis for the master branch?
This prevents executing travis for all PR that target master branch

Reply:
Oh, sorry. I forgot to restore it as it is.
I was using Travis CI to check some tests before, as those tests couldn't be run on my laptop. I'll fix it immediately.